src/components/App.css

```diff
@@ -3,19 +3,36 @@
   margin: 0;
```

Comment:
- [ ] Let's try to make a couple of subtle changes to make it look a lot more like the original design.

1. Let's add a white bg to the Navbar and a border-bottom gray.
2. Let's change the weight the fonts of the fonts on the Navbar. Title is Montserrat-Bold, you instead applied Montserrat-Black
3. Let's increase the padding vertical a little. We can also center the items making use of Flex.
4. You can also increase a little bit the horizontal padding of the content (cards and form).

Yours:
![image](https://user-images.githubusercontent.com/79114233/145693050-2fa971d9-3665-4e7c-9051-d80172da08e4.png)

Template:
![image](https://user-images.githubusercontent.com/79114233/145693042-c3218989-415c-48e7-a9ae-0fa95594d773.png)

If you have followed mi indications until here, you should end with a more accurate design than the one provided.

Yours:
![image](https://user-images.githubusercontent.com/79114233/145693363-1cab827a-5e03-42bb-aabc-d4111b41fb93.png)


After a few minor twerks:
![image](https://user-images.githubusercontent.com/79114233/145693351-ab3846f0-d66d-43c2-bd32-46e4be938be3.png)

Reply:
Done! Thanks for pointing them out, I have fixed them.